.github/workflows/release-porting-guide.yml

```
@@ -0,0 +1,61 @@
+---
+name: Ansible porting guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: Release version. For example, 11.1.0
+        required: true
+      ansible-major-version:
+        description: Major version. For example, 11
+        required: true
+      ansible-build-data-branch:
+        description: Release Branch name from the Ansible Build data PR
+        required: true
+
+jobs:
+  upload-porting-guide:
+    name: Extract the porting guide
+    runs-on: ubuntu-latest
+    env:
+      PORTING_GUIDE_RST: porting_guide_${{ inputs.ansible-major-version }}.rst
+      PORTING_GUIDE_RST_PATH: "./ansible-build-data/${{ inputs.ansible-major-version }}/porting_guide_${{ inputs.ansible-major-version }}.rst"
+      ANSIBLE_RELEASE_BRANCH: ${{ inputs.ansible-build-data-branch }}
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check out ansible-build-data
+        uses: actions/checkout@v4
+        with:
+          repository: ansible-community/ansible-build-data
+          ref: ${{ inputs.ansible-build-data-branch }}
+          path: ansible-build-data
+
+      - name: Copy the RST file to the correct path
+        run: cp -v "${PORTING_GUIDE_RST_PATH}" docs/docsite/rst/porting_guides/
+
+      - name: Set up git
+        run: |
+          git switch --create "${GIT_BRANCH}"
+          ACTOR_NAME="$(curl -s https://api.github.com/users/"${GITHUB_ACTOR}" | jq --raw-output '.name // .login')"
+          git config --global user.name "${ACTOR_NAME}"
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
+
+      - name: Add the porting guide
+        run: git add docs/docsite/rst/porting_guides/"${PORTING_GUIDE_RST}"
+
+      - name: Commit the porting guide
+        run: >-
+          git diff-index --quiet HEAD ||
+          git commit -m "${CI_COMMIT_MESSAGE}"
+
+      - name: Push to the repo
+        run: git push origin "${GIT_BRANCH}"
+
+      - name: Create the porting guide PR
```

Isn't a token needed for this API call?
```suggestion
      - name: Create the porting guide PR
        env:
          GH_TOKEN: ${{ github.token }}
```